packages/migrate/src/__tests__/DbPull/cockroachdb.test.ts

```diff
@@ -0,0 +1,263 @@
+// describeIf is making eslint unhappy about the test names
+/* eslint-disable jest/no-identical-title */
+
+import { jestConsoleContext, jestContext, jestProcessContext } from '@prisma/get-platform'
+import path from 'path'
+
+import { DbPull } from '../../commands/DbPull'
+import { setupCockroach, tearDownCockroach } from '../../utils/setupCockroach'
+import { SetupParams, tearDownPostgres } from '../../utils/setupPostgres'
+
+const isMacOrWindowsCI = Boolean(process.env.CI) && ['darwin', 'win32'].includes(process.platform)
+if (isMacOrWindowsCI) {
+  jest.setTimeout(60_000)
+}
+
+const describeIf = (condition: boolean) => (condition ? describe : describe.skip)
+
+const ctx = jestContext.new().add(jestConsoleContext()).add(jestProcessContext()).assemble()
+
+// To avoid the loading spinner locally
+process.env.CI = 'true'
+
+const originalEnv = { ...process.env }
+
+describeIf(!process.env.TEST_SKIP_COCKROACHDB)('cockroachdb', () => {
+  if (!process.env.TEST_SKIP_COCKROACHDB && !process.env.TEST_COCKROACH_URI_MIGRATE) {
+    throw new Error('You must set a value for process.env.TEST_COCKROACH_URI_MIGRATE. See TESTING.md')
+  }
```

Comment:
I think this was how it was before too but it still reads weirdly that we already discriminate on `!process.env.TEST_SKIP_COCKROACHDB` in the `describeIf()` but then do it again in the `if`

Reply:
Oh yes, this is weird but nothing new here. That code does run somehow if the env vars are not set 🙈 (which in theory should not happen)